contracts/SpokePool.sol

```diff
@@ -137,14 +137,18 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
      * @param _hubPool Hub pool address to set. Can be changed by admin.
      * @param _wrappedNativeTokenAddress wrappedNativeToken address for this network to set.
      * @param timerAddress Timer address to set.
+     * @param eip712DomainName User readable name of the signing domain, i.e. the name of the DApp or the protocol.
+     * @param eip712DomainVersion The current major version of the signing domain.
      */
     constructor(
         uint32 _initialDepositId,
         address _crossDomainAdmin,
         address _hubPool,
         address _wrappedNativeTokenAddress,
-        address timerAddress
-    ) Testable(timerAddress) {
+        address timerAddress,
+        string memory eip712DomainName,
+        string memory eip712DomainVersion
+    ) Testable(timerAddress) EIP712CrossChain(eip712DomainName, eip712DomainVersion) {
```

Comment:
nit: I think you should pass in the hardcoded `eip712DomainName` and `eip712DomainVersion` here rather than have to change the constructor of Arbitrum/Ovm/PolygonSpokePool contracts. They all would use the same name and version

Reply:
changed here https://github.com/across-protocol/contracts-v2/pull/222/commits/e8a9c8d006e0d444dce07c329181740e774d8c13